text/src/autogluon/text/automm/data/process_text.py

```diff
@@ -260,13 +314,17 @@ def build_one_token_sequence_from_text(
             "Token indices sequence length is longer than.*result in indexing errors"
         )
         for col_name, col_text in text.items():
+            # augment text if specify in config
+            if (is_training and (len(col_text.split(' ')) >= 5)):
+                 # naive way detect text/categorical/numerical: more than 5 words
```

Comment:
Should we put it as part of the config?

Reply:
I think augmentation should be applied on to text column, not categorical/numerical columns even though they are fed into text backbone. I will put the length for detection into config.